### PR TITLE
Corrected the convert in Numeric Popup dialog

### LINKED ISSUE #7293: Correct the convert to numeric popup dialogue
I hope @N-thony can do this, but it may need input from @shadrackkibet as well as a check that my suggestions below are correct.

When a factor (possibly also a labelled character) is converted to numeric, then there is usually a popup if the factor contains a mix of numeric and non-numeric values.  But if all is non-numeric, and it is unlabelled, then R-Instat currently simply uses the ordinal values, and makes them into labels!
To illustrate us the fert data from survey.  Make it into a factor.
a) Then recode into fert_recoded by just making 0 into None and 3 into Max.  The recoding doesn't have labels now - which is fine!
b) Also recode by the above, but make all the other levels into Some.  (So nothing is numeric in fert_recoded1)
Note that neither of them is now labelled.

Now right-click and make a) into Numeric.  The popup allows the option of the Labelled convert.  That is even though there are now no labels!  If you accept, then it adds labels again - which is fine - except it may not be what is wanted.  It does what I would call "Ordinal Convert".  I suggest we simply change Labelled Convert to `Ordinal Convert `in the popup.  Maybe the word Ordinal is in a Colour, so it stands out.
When nothing is numeric, and you convert to numeric it misses the check and uses the ordinals and makes it numeric.  This should be trapped and the same dialogue given.  It should again say Ordinal Convert.

### PR DESCRIPTION
**Pull request by N-thony**
Fixes #7293
I have added the checking when nothing is a numeric in the column to convert to numeric and also changed `Labelled Convert` to `Ordinal Convert`
@africanmathsinitiative/developers @rdstern this is ready for review. Thanks.

### CODE REVIEW

**Review by rdstern — CHANGES_REQUESTED** [review]
@N-thony almost I think.  As discussed use Labelled convert - as before - when labelled, and ordinal convert when there are no labels.

**Review by rdstern — APPROVED** [review]
Looks good now

File: instat/frmConvertToNumeric.vb [hunk 1]
```diff
@@ -30,6 +30,16 @@
         strDataFrameName = strDataFrame
     End Sub
 
+    Public Sub CheckLabels(bCheck As Boolean)
+        If bCheck Then
+            cmdLabelledConvert.Text = "Lebelled Convert"
```
Comment (lloyddewit) [inline instat/frmConvertToNumeric.vb:35]:
```suggestion
            cmdLabelledConvert.Text = "Labelled Convert"
```

File: instat/frmConvertToNumeric.vb [hunk 2]
```diff
@@ -30,6 +30,16 @@
...
+            lblOrdinal.Visible = False
```
Comment (lloyddewit) [inline instat/frmConvertToNumeric.vb:36]: Move this out of the `if else` block and set it to `lblOrdinal.Visible = Not bCheck`?

File: instat/Model/RCommand/clsPrepareFunctionsForGrids.vb [hunk 3]
```diff
@@ -150,6 +150,27 @@ Public Class clsPrepareFunctionsForGrids
...
     End Sub
+
+    ''' <summary>
+    '''  Check if the column factor contains labels.
+    ''' </summary>
+    Public Function CheckHasLabels(strColumnName As String) As Boolean
+        Dim clsColmnLabelsRFunction = New RFunction
+        Dim clsGetColumnsFromData As New RFunction
+
+
```
Comment (lloyddewit) [inline instat/Model/RCommand/clsPrepareFunctionsForGrids.vb:161]:
```suggestion
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@N-thony thanks, looks good, just a couple of small questions

**Comment by N-thony** [comment]
@lloyddewit I resolved your comments. Please take a look. Thanks.

**Review by rdstern — APPROVED** [review]
Still looks good to me

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**